protocols/request-response/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-request-response"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Generic Request/Response Protocols"
-version = "0.25.0"
+version = "0.26.0"
```

Comment:
Why would this be a breaking change? I don't think we are exposing `serde_cbor` anywhere, right?